pyomo/contrib/doe/result.py

```diff
@@ -27,6 +27,7 @@
 
 
 from pyomo.common.dependencies import numpy as np, pandas as pd, matplotlib as plt
+from pyomo.core.expr.numvalue import value
```

Comment:
This seems unused in the new changes. Is this necessary?

Reply:
Ah, this is a sneaky bug fix. When I opened this file, my IDE started yelling loudly that `value` was used but never imported. (Also indicates that we have untested code in here - not good!)